### ISSUE TITLE
Iptables rules are not getting applied if removed completely

### ISSUE THREAD

**Issue by kushaldas** [freedomofpress/securedrop#5309]
## Steps to Reproduce

- [ ] On `app` run `iptables -F`
- [ ] `./securedrop-admin install` on tails again
- [ ] reboot app

## Expected Behavior

- The rules should come back on `app`

## Actual Behavior

- No rules even after reboot via `iptables -L`

## Comments

Suggestions to fix, any other relevant information.

**Comment by conorsch** [comment]
Cannot reproduce. I tested using staging VMs in Qubes, running the following commands on `app-staging`:

```
sudo iptables -P INPUT ACCEPT
sudo iptables -P OUTPUT ACCEPT
sudo iptables -P FORWARD ACCEPT
sudo iptables -F
sudo iptables -X
sudo iptables -S # to confirm the rules were purged
```

I then rebooted the `app-staging` VM, logged in again, and ran `sudo iptables -S`, which showed that all the rules had been restored. I did _not_ run playbooks against the server as described in the initial report—a reboot was sufficient to reload the rules back to baseline.
